src/app/components/tile/index.html

```diff
@@ -175,4 +175,57 @@
     </sky-demo-page-code>
   </sky-demo-page-example>
 
+  <sky-demo-page-content sectionHeading="UX guidelines">
+    <p>
+      Tiles are containers to divide and group a large number of categories of varied content on a page. Use tiles when you have to provide access to a large number of content groupings where users would benefit from the capabilities to reorder, expand/collapse the tiles, or have personalization options for the content displayed.
+    </p>
+    <p>
+      See <a routerLink="/design/guidelines/content-containers">content container guidelines</a> for more details.
+    </p>
+  </sky-demo-page-content>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+      <p>
+        Use tiles to divide a page with many categories of content, especially when the format or presentation of the content will differ across the categories.
+      </p>
+    </stache-column>
+    <stache-column screenSmall="6">
+      <div class="screenshot screenshot-success"><img src="~/assets/img/guidelines/tile/tile-example.png" style="width: 100%;" /></div>
+    </stache-column>
+  </stache-row>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+      <p>
+        Don't use tiles to divide a page into a small number of sections.
+      </p>
+    </stache-column>
+    <stache-column screenSmall="6">
+      <div class="screenshot screenshot-danger"><img src="~/assets/img/guidelines/tile/bad-example-1.png" style="width: 100%;" /></div>
+    </stache-column>
+  </stache-row>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+      <p>
+        Don't use tiles as a container for all content on a page outside of the summary.
```

Comment:
I wonder if we really need this guideline. I mean, isn't it implicit from the previous guideline? No harm in including it, I guess, but I wonder if it's worth asking Todd if we really want this guideline.  

I think this might be a scenario where we want to use singular instead of plural. What do you think of changing this to something like "Don't use a single tile to contain all of a page's content after the summary."?

Reply:
Checked in with Todd about this, and he said it's worth including because even though it may be redundant, the misuse is common.